src/console_cmd.c

```diff
@@ -446,6 +654,74 @@ static TbBool script_set_pool(PlayerNumber plyr_idx, const char *creature, const
   return true;
 }
 
+long get_creature_model_for_command(char *msg)
+{
+    long rid = get_rid(creature_desc, msg);
+    if (rid >= 1)
+    {
+        return rid;
+    }
+    else
```

Comment:
I mentioned this yesterday on discord, but it would be really nice if for this, and all future console commands, we could accept a . instead of an _ in the console commands, at least until _ actually works. So, bile.demon for bile_demon, and poison.gas for poison_gas.
See [here](https://github.com/dkfans/keeperfx/pull/926/files#diff-a76846cfc41816952cce3bb582bc96872f2b649eb530c54aee51626480602e68R248).